rdf-abac-core/src/main/java/io/telicent/jena/abac/attributes/syntax/tokens/TokenizerABAC.java

```diff
@@ -519,7 +538,7 @@ private void readNumber() {
                 token.setType(TokenType.HEX);
                 return;
             }
-        } else if ( ch == '-' || ch == '+' ) {
+        } else if ( ch == CH_MINUS || ch == CH_PLUS ) { // unreachable code?
```

Comment:
Yeah it looks like the `parseNumeric()` method (which you also cleaned up in this PR) already handles the `+` or `-` character prior to calling this method, so unless this method is called from anywhere else I think this branch is probably defunct

Reply:
Thanks for confirming. Removed.